index.html

```diff
@@ -1,75 +1,72 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="style.css" />
-  <link
-      href="https://fonts.googleapis.com/icon?family=Material+Icons"
-      rel="stylesheet"
-    />
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
   <title>Morrighan's Portfolio</title>
 </head>
+
 <body>
-   <!-- start toolbar -->
+  <!-- start toolbar -->
   <header class="bg_white df">
     <div class="logo_placeholder df">
-      <a href="index.html" class="logo xs-transparent_btn primary_b"
-        >My Logo</a>
+      <a href="index.html" class="logo xs-transparent_btn primary_b">My Logo</a>
     </div>
     <nav>
-      <a
-      href="#menu"
-      class="btn xs-transparent_btn df btn_fc"
-      ><i class="material-icons primary_b">menu</i></a
-    >
+      <a href="#menu" class="btn xs-transparent_btn df btn_fc menu_btn">
+        <i class="material-icons primary_b">menu</i>
+      </a>
+      <ul class="menu">
+        <li class="btn xs-transparent_btn df btn_fc">
+          <a href="#Portfolio">Portfolio</a>
+        </li>
+        <li class="btn xs-transparent_btn df btn_fc">
+          <a href="#About">About</a>
+        </li>
+        <li class="btn xs-transparent_btn df btn_fc">
+          <a href="#Contact">Contact</a>
+        </li>
+      </ul>
     </nav>
   </header>
   <!-- end toolbar -->
 
   <!-- start main -->
   <main class="shapes_bg bg_white df">
```

Comment:
- [ ] Kindly wrap all your sections outside the `main` within the `main` tag, this will make your code more semantic and easy to read for tools like screen readers. These sections are also a part of the main content of your web page.